src/coreclr/jit/codegenarm.cpp

```diff
@@ -550,7 +550,7 @@ void CodeGen::genLclHeap(GenTree* tree)
         inst_JMP(EJ_lo, done);
 
         // Update SP to be at the next page of stack that we will tickle
-        GetEmitter()->emitIns_R_R(INS_mov, EA_PTRSIZE, REG_SPBASE, regTmp);
+        GetEmitter()->emitIns_Mov(INS_mov, EA_PTRSIZE, REG_SPBASE, regTmp, /* canSkip */ false);
```

Comment:
Many of these are currently `/* canSkip */ false` because that's how the existing code handled it.

However, almost all of them except for `genIntToIntCast` are technically elidable (assuming the registers are the same) as they don't do have any special semantics.